e2e/ansible/playbooks/hyperconverged/test-replica-node-affinity/replica-node-affinity-vars.yml

```diff
@@ -0,0 +1,18 @@
+---
+test_name: verify_replica_deployment_with_node_affinity
+
+patch_file: replica_patch.yml
+
+create_sc: storage-class.yaml
+
+percona_link: https://raw.githubusercontent.com/openebs/openebs/master/k8s/demo/percona/demo-percona-mysql-pvc.yaml
+
+replica_count: 2
```

Comment:
If the test involves editing/updating replica count values, please obtain the initial replica count based on available Kubernetes host nodes (refer cassandra scale or replica fail tests)

Reply:
Included this considering we have 3 nodes in our CI setup

Reply:
- Obtaining the number of nodes and assigning replica count by subtracting  the node count by one.